Godeps/Godeps.json

```diff
@@ -4071,6 +4067,14 @@
 		{
 			"ImportPath": "vbom.ml/util/sortorder",
 			"Rev": "db5cfe13f5cc80a4990d98e2e1b0707a4d1a5394"
+		},
+		{
+			"ImportPath": "golang.org/x/net/http/httpguts",
+			"Rev": "c39426892332e1bb5ec0a434a079bf82f5d30c54"
+		},
+		{
+			"ImportPath": "golang.org/x/net/internal/socks",
+			"Rev": "c39426892332e1bb5ec0a434a079bf82f5d30c54"
```

Comment:
Got it. Let me do the same in other Godep.json files

Reply:
I would imagine other Godep.json files should be handled fine by the script? It's more of the actual source code who has dependency on those files, you probably need to go in there and modify them. Does the code compile after you run the script?

Reply:
ping @wenjiaswe

Reply:
@PrasadG193 Sorry for the delay! I was busy with KubeCon and just caught up with this. Actually, what you need to do is to update files in https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io in this PR, instead of vendors. cc @dims for confirmation.

Reply:
@wenjiaswe we have already updated dependencies in staging/src/k8s.io/* files in this PR. But the `hack/verify-godeps.sh` script is failing as repos in vendor/k8s.io/* are still using older version of `golang.org/x/net/...` packages

Reply:
@PrasadG193 please run `hack/update-staging-godeps.sh` and also `hack/update-godep-licenses.sh`